admin/models/check.js

```diff
@@ -34,6 +34,9 @@ const Check = new Schema({
     type: Date,
     required: true
   },
+  checkStartedAt: {
```

Comment:
this needs adding to the db schemas too

Reply:
On the commit 6753b94df112d59e2f8c74810e3c513853d7fc5a you can see the change under design/schemas/database/checks.json
Did I miss something?